tests/migrate/test_plural.py

```diff
@@ -74,6 +75,55 @@ def test_plural_too_many_variants(self):
         )
 
 
+class TestPluralOrder(MockContext):
+    # Plural categories corresponding to Lithuanian (lt).
+    plural_categories = ('one', 'few', 'other')
+
+    def setUp(self):
+        self.strings = parse(PropertiesParser, '''
+            # These forms correspond to (one, other, few) in CLDR
+            deleteAll = Pašalinti #1 atsiuntimą?;Pašalinti #1 atsiuntimų?;Pašalinti #1 atsiuntimus?
+        ''')
+
+        self.message = FTL.Message(
+            FTL.Identifier('delete-all'),
+            value=PLURALS(
+                'test.properties',
+                'deleteAll',
+                EXTERNAL_ARGUMENT('num')
+            )
+        )
+
+    def test_common_order(self):
+        self.assertEqual(
+            evaluate(self, self.message).to_json(),
+            ftl_message_to_json('''
+                delete-all =
+                    { $num ->
+                        [one] Pašalinti #1 atsiuntimą?
+                        [few] Pašalinti #1 atsiuntimų?
+                       *[other] Pašalinti #1 atsiuntimus?
```

Comment:
This isn't the right order. `few` should end with `atsiuntimus` and `other` should end with `atsiuntimų`. I don't think it's a good idea to reuse these translations to test different orders.

I think it's sufficient to make this test class about weird ordering. If you wish, we could add another test class for testing proper order, but admittedly this is already tested by `TestPlural`.